platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -361,6 +363,12 @@
 		.de("Lokaler Name");
 	I18n0 LOCAL_PORT = new I18n0("Local Port")//
 		.de("Lokaler Port");
+	I18n0 LOCALIZATION = new I18n0("Localization")//
+		.de("Lokalisation");
```

Comment:
I think this should be `Lokalisierung` (much like above and below)

Reply:
done